huaweicloud/utils/utils.go

```diff
@@ -556,3 +556,32 @@ func IsUUID(uuid string) bool {
 	match, _ := regexp.MatchString(pattern, uuid)
 	return match
 }
+
+// The method can used to remove empty elements from an array
+func RemoveEmptyStrings(s []interface{}) []interface{} {
+	result := make([]interface{}, 0, len(s))
+	for _, elem := range s {
+		if isNonEmpty(elem) {
+			result = append(result, elem)
+		}
+	}
+	return result
+}
+
+// The method can used to check element is empty or not
```

Comment:
```suggestion
// The method can be used to check element is empty or not.
```
There are syntax issues present, please make adjustments in similar areas